### ISSUE TITLE
FAC16 mentor discussion mintues | 11th Feb

### ISSUE THREAD

**Issue by onwordi** [foundersandcoders/master-reference#940]
Mentoring Group Discussion

`Matt` - We're going to have a discussion about mentoring, what it takes to be a mentor and talk about mentoring stuff... tips and tricks.

Everyone has different ways of learning, so its good to hear different mentoring techniques from everyone.

If everyone goes round and says what they appreciated in mentors

`Lucy`: couple of weeks were mentors were more hands on and some who would wait on the side for us to ask for help. It was nice to have a mentors who were a bit more proactive to stop us from going down a rabbit hole.

`Oliver`: i agree, it felt like the mentors were more invested when they were more proactive.

`Kristina`: i definitely agree, but it sometimes felt overwhelming when overloaded with too much content.

`Sandra`: i kind of disagree, When someone is standing over me unannounced, its stressful and i feel like i cant type.

Martin: its easy to give help to those who ask for help, and easy to not give help when people don’t ask

`Sak`: One thing i appreciate All mentors are all so different, and its nice to see the different personalities coming together.

`Kristina`: maybe ask the class how they like to be mentored

`Charlie`: some people don't like the pressure and people are fine with it.

`Hannah`: sometimes i don’t realise i need help, so having someone passively come over and give advice is really helpful

`Charmine`: maybe just ask someone 'where are you at?' it can be helpful

`Sak`: sofia asked you to explain, which lets you know where you're at.

`Charlie`: this can be helpful

`Kristina`: the fact she wrote the tutorials she was able to explain the well

`Hannah`: its really nice when the mentors are 5/10mins early, its nice to see and enough to inspire confidence.

`Sak`: initially when you start its hard to ask for help. If you're one of the early mentors be more engaging so the new cohort feel more at ease to ask, so by the later weeks they are definitely more comfortable to ask

`Matt`: Lets talk about traits, in one word

Enthusiasm, Openness, Calm, Direct, Observant, being ok with failure/breaking things, will to help, committed to understanding, approachable, grounded, really prepared, diliengce, concise, fun, patience 

`Charmine`: Don't assume knowledge

`Martin`: but it can go the other way

`Cynthia`: but you can just say 'Does everyone know what destructuring is?...no? ok.. destructuring is...'

`Kristina`: assertive with the class, sometimes the class would be chatty, and it would take someone in the front of the class to tell people 'hey, lets be quiet' and not the mentor

`Hannah`: I really liked it when the mentor brought in biscuits for everyone

`Jason`: people do things in different ways, and not everyone can understand the same ways. 

`Matt`: I agree.. i like to try different ways so that the individual get it. Use a whiteboard, paper, verbal

`Oliver`: i agree, its really nice to hear it in different ways

`Sak`: i think when you're pair programming, get the other pair to try and explain to that they can both grow.

`Hannah`: its also important to give the other person the benefit of the doubt that you're explaining things well

`Cynthia`: respect that some people may not be ready to explain their full understanding, or may feel pressure at that time, and have the right to say they're not comfortable explaining themselves at that precise time when questioned. Give them time and approach in a different way.

`Sandra`: sometimes you may get someone else's explanation better than others. 

`Hannah`: even if you revise everything and prepare you may still not be the best mentor and thats perfectly ok.

`Matt`: Say one thing you're worried about mentoring (big or small) and we'll talk about them

`Jason`: presenting, because presenting in front of a client can be nerve wrecking. You're not being judged, its a different environment. I don’t think I’m very good at public speaking.

`Susan`: if the chrome cast is going to fail

`Martin`: dealing with peoples operating systems and installation

`Matt`: its ok to say you just don’t know. They should be able to pair with someone who’s laptop is working ad continue with the work, and the help them afterwards so that the class isn’t disruptive

`Kristina`: having a note of FAQ or vocab, as you're preparing write them down.

`Charlie`: after each week, we're meant to make notes on the HackMd. Before each week, when we're preparing the week, I can say what people were stuck on and what needs more work on. 

`Matt`: its used to gauge were people are and when they got stuck. Theres also an MD notes mentors can look at to see how to prepare for their week

`Sandra`: i think I’m worried about not being able to explain things properly

`Oliver`: a big part of that is coming to terms with the fact that you wont be expected to know everything

`Martin`: You can also remind the class that you went through the same thing 16 weeks ago.

`Kristina`: sometimes the mentor would ask me "have you googled that" which can give time to distract as you find out the answer yourself.

`Charmine`: i rather the mentor saying "give me 5mins and ill get back to you" instead on muddling through and messing up my current understanding.

`Matt`: running through all the workshops a few times before your mentoring week, a couple of times will be helpful. When you can tell a mentor is unprepared is not the greatest

`Sak`: Im concerned ill help the good people, and they'll get better and better... and the others may not get my attention. Its easy to help the ones in top side. Its easy to help the good people as a teacher.

`Hannah`: pair programming really helps this, so that no one is excelling by themselves. As a mentor you have authority to say switch pairs.

`Charmaine`: Also some days people may seem above the class, and other days they may not be.

`Hannah`: I’m worried that when i explain, ill either be too opaque or patronising. 

`Charlie`: tone of voice is something to be aware of. If you sound patronising, then it will go to a negative way.

`Charmine`: i think asking questions is a good start so that you know how to help and where/what to explain

`Oliver`: its good to keep in mind all the user manuals, which will help to frame everything mentally beforehand.
